libpod/container.go

```diff
@@ -237,6 +237,9 @@ type ContainerNamedVolume struct {
 	Dest string `json:"dest"`
 	// Options are fstab style mount options
 	Options []string `json:"options,omitempty"`
+	// SetAnonymous sets the named volume as anonymous even if it has a name
+	// This is used for emptyDir volumes from a kube yaml
+	SetAnonymous bool `json:"setAnonymous"`
```

Comment:
Minor nit: Maybe `IsAnonymous` instead of `SetAnonymous`? And probably set `omitempty`

Reply:
fixed